_config.yml

```diff
@@ -25,13 +25,13 @@ social:
 # This is Search Engine Optimization. It'll help your website look better on search.
 seo:
   title:       "uOttaHack: uOttawa's Inaugural Hackathon"
```

Comment:
Perhaps we should update this line to also say MLH

Reply:
For example "uOttaHack: uOttawa's Inaugural MLH Hackathon"

Reply:
Oops, missed that